### ISSUE TITLE
Quay Migration tool example 

### ISSUE THREAD

**Issue by wcampos** [jfrog/docker2artifactory#5]
The example for quay is hard to follow up, it took some time to reverse engineer what you guys were expecting for us to pass.. Should add an example similar to the below example into the Readme.md files. I tried to push a hotfix branch but had no permissions. 

#### Example

* Create a file `images.txt`
  * content of that file should contain `<namespace>/<image_name>:<tag>` (tag is optional)
  * Example will pull 
>> Make sure you replace <namespace> and <image_name> with real values
```bash
cat images.txt
<namespace>/<image_name>:latest 
```

* Now run your command 
```bash
python DockerMigrator.py quay --num-of-workers 4 -v --image-file ./images.txt <namespace> <token> https://<af-server-name>:<af-server-port>/artifactory <af-admin-name> <af-admin-user-passwd> <af-repo-name>
```
>> Admin user for Artifactory will help with repo permissions. If you are not using admin, make sure the user had admin access to the destination repository

**Comment by jfrog-aparicio** [comment]
Hello William, you can submit a pull request by first forking the project, creating a branch on that fork, pushing the changes and then submitting a pull request.
